packages/shared-state/files/www/cgi-bin/shared-state

```diff
@@ -27,3 +27,11 @@ PATH_INFO="${PATH_INFO#/}"
 echo "$PATH_INFO" | grep -q '^[a-zA-Z0-9_-]*$' || exit 22
 
 cat - | shared-state reqsync "$PATH_INFO"
```

Comment:
shouldn't this line go away?

Reply:
Yes, sorry, problem arised when I lost the change and then copy pasted a diff I had and then I did not retested ...